shared/src/business/useCaseHelper/stampDisposition/addDraftStampOrderDocketEntryInteractor.ts

```diff
@@ -88,6 +88,7 @@ export const addDraftStampOrderDocketEntry = async (
       filedBy: user.judgeFullName || user.name,
       freeText: `${originalDocketEntryEntity.documentType} ${formattedDraftDocumentTitle}`,
       isDraft: true,
+      isFileAttached: true,
```

Comment:
I believe this is the fix from a few weeks ago for stamp order crashing. I'd love to see the `stamp-order` cypress test expanded so that bug would have been caught in CI 😄

Reply:
Letting you know this has been done. Check out the changes to `stamp-order.cy.ts`

Reply:
Thank you!